catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet

```diff
@@ -0,0 +1,3426 @@
+local id = 'projects/neon-prod-earthengine/assets/HSI_REFL/001';
+local subdir = 'neon-prod-earthengine';
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local license = spdx.proprietary;
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+{
+  // TODO(google): Remove when the dataset is ready.
+  'gee:skip_indexing': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'NEON Directional Surface Reflectance',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The NEON AOP surface directional reflectance data is an orthorectified 
+    hyperspectral VSWIR (visible to shortwave infrared) data product, containing 
+    426 bands spanning wavelengths from ~380 nm to 2510 nm. Wavelengths 
+    between 1340-1445 nm and 1790-1955 nm are set to -100; these are water vapor 
+    absorption bands without valid values. The dataset also contains additional 
+    QA and raster bands (B427 - B442). Data are collected by one of three NEON 
+    Imaging Spectrometer units, AVIRIS-NG sensors built by NASA's Jet Propulsion 
+    Lab (JPL). Major corrections include calibration, orthorectification, and 
+    atmospheric correction. Reflectance is converted to a mosaic from flightlines 
+    using the nadir-most pixels from the flight lines observed with the least 
+    cloud cover. Spatial resolution is 1 m and spectral resolution is ~5 nm. 
+    See [NEON Data Product
+    DP3.30006.001](https://data.neonscience.org/data-products/DP3.30006.001) for
+    more details.
+    Documentation: [NEON DP3.30006.001 Spectrometer orthorectified surface directional reflectance - mosaic Quick Start
+    Guide](https://data.neonscience.org/api/v0/documents/quick-start-guides/NEON.QSG.DP3.30006.001v1?inline=true&fallback=html)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'airborne',
+    'canopy',
+    'ecology',
+    'forest',
+    'hyperspectral',
+    'neon',
+    'reflectance',
+    'surface-reflectance',
```

Comment:
the `surface-reflectance` keyword is failing with this message: "'keyword must contain only lowercase letters, digits, and underscores and be at most 49 characters long: "surface-reflectance"', IssueLevel.error)", as well as an "only one instance of" error, but I see it in the GEE dataset tags here: https://developers.google.com/earth-engine/datasets/tags/surface-reflectance.